graphql/course/mutations.ts

```diff
@@ -32,7 +32,7 @@ class PublicCourseCreateInput {
     allowContact?: boolean;
 
     @TypeGraphQL.Field((_type) => CourseSubject, { nullable: true })
-    subject?: CourseSubject;
+    subject!: CourseSubject;
```

Comment:
I don't think the _definite assignment assertions_ are necessary here, just subject: should be fine